src/micromamba/constructor.cpp

```diff
@@ -16,13 +17,23 @@ using namespace mamba;  // NOLINT(build/namespaces)
 void
 init_constructor_parser(CLI::App* subcom)
 {
-    subcom->add_option("-p,--prefix", constructor_options.prefix, "Path to the prefix");
+    auto& config = Configuration::instance();
+
+    auto& extract_conda_pkgs
+        = config.insert(Configurable("constructor_extract_conda_pkgs", false)
+                            .group("cli")
+                            .rc_configurable(false)
+                            .description("Extract the conda pkgs in <prefix>/pkgs"));
     subcom->add_flag("--extract-conda-pkgs",
-                     constructor_options.extract_conda_pkgs,
-                     "Extract the conda pkgs in <prefix>/pkgs");
-    subcom->add_flag("--extract-tarball",
-                     constructor_options.extract_tarball,
-                     "Extract given tarball into prefix");
+                     extract_conda_pkgs.set_cli_config(0),
+                     extract_conda_pkgs.description());
+
+    auto& extract_tarball = config.insert(Configurable("constructor_extract_tarball", false)
+                                              .group("cli")
+                                              .rc_configurable(false)
+                                              .description("Extract given tarball into prefix"));
+    subcom->add_flag(
+        "--extract-tarballs", extract_tarball.set_cli_config(0), extract_tarball.description());
```

Comment:
this flag has _changed_??

It's used in the miniforge constructor project and this will definitely break things.

Reply:
fixed, sorry for that